BGL/doc/BGL/Concepts/HalfedgeGraph.h

```diff
@@ -55,5 +55,11 @@ Expression                              | Returns
 `next(h, g)`                            | `halfedge_descriptor`                                                        | The next halfedge around its face.
 `prev(h, g)`                            | `halfedge_descriptor`                                                        | The previous halfedge around its face.
 `boost::graph_traits<G>::%null_halfedge()` | `halfedge_descriptor`                                                     | Returns a special halfedge that is not equal to any other halfedge.
+
+\cgalHeading{Invariants}
+
+`halfedge(edge(h,g))==h`
```

Comment:
Similarly as the BGL concept IncidentEdge. An out-edge of a vertex `v` must guarantee that the source vertex is the vertex `v`. However if you consider the two out-edges from the vertex of any edge, the edges are equal while they obviously won't provide the same source vertex.